addons/auth_totp/views/user_preferences.xml

```diff
@@ -1,51 +1,64 @@
 <odoo>
-    <record model="ir.ui.view" id="view_totp_list">
-        <field name="name">users list: add totp status</field>
+    <record id="res_users_view_search" model="ir.ui.view">
+        <field name="name">res.users.view.search.inherit.auth.totp</field>
         <field name="model">res.users</field>
-        <field name="inherit_id" ref="base.view_users_tree"/>
+        <field name="inherit_id" ref="base.view_users_search" />
         <field name="arch" type="xml">
-            <tree>
-                <field name="totp_enabled"/>
-            </tree>
+            <search string="Users">
```

Comment:
Are you erasing it ? Shouldn't it be in an xpath ?

Reply:
this adds automatically the content a the end of the search view. But I can see you are not a big fan of shortcuts like this. So i'll go for an xpath.